CONTRIBUTING.md

```diff
@@ -78,17 +78,17 @@ Check out our dedicated [Bevy Organization document](/docs/the_bevy_organization
 
 Our merge strategy relies on the classification of PRs on two axes:
 
-* How controversial are the design decisions
-* How complex is the implementation
+* How controversial are the design decisions.
+* How complex is the implementation.
 
 Each [label](https://github.com/bevyengine/bevy/labels) has a prefix denoting its category:
 
-* A: Area (e.g. A-Animation, A-ECS, A-Rendering)
-* C: Category (e.g. C-Breaking-Change, C-Code-Quality, C-Docs)
-* D: Difficulty (e.g. D-Complex, D-Good-First-Issue)
-* O: Operating System (e.g. O-Linux, O-Web, O-Windows)
-* P: Priority (e.g. P-Critical, P-High)
-* S: Status (e.g. S-Blocked, S-Controversial, S-Needs-Design)
+* A: Area (e.g. A-Animation, A-ECS, A-Rendering).
+* C: Category (e.g. C-Breaking-Change, C-Code-Quality, C-Docs).
+* D: Difficulty (e.g. D-Complex, D-Good-First-Issue).
+* O: Operating System (e.g. O-Linux, O-Web, O-Windows).
+* P: Priority (e.g. P-Critical, P-High).
+* S: Status (e.g. S-Blocked, S-Controversial, S-Needs-Design).
```

Comment:
I wouldn't use periods here. Bulleted list items should generally use periods if they either contain a full sentence or complete the introductory stem sentence (the sentence before ":"), which is not the case here. Stylistically, ")." also doesn't look as clean in my opinion.